ee/vellum_cli/push.py

```diff
@@ -80,6 +80,9 @@ def push_command(
         module_dir = workflow_config.module.replace(".", os.path.sep)
         for root, _, files in os.walk(module_dir):
             for filename in files:
+                if not filename.endswith(".py"):
+                    continue
```

Comment:
Pretty sure that would be going to local disk not the virtual file system for modules

Reply:
Uh no anything going into the route is just kept in memory or a virtual file module loader, any files would have to be loaded from the docker image currently. My initial gut reaction to implementing this is a 5🥴 out of 5🥴 since it might involve weird monkeypatching since it needs to be able to access the virtual files and files on the docker image